mozanalysis/bootstrap.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from functools import partial
+
+
+def bootstrap(iteration, stat_fn, broadcast_data):
+    np.random.seed()
+    n = len(broadcast_data.value)
+    randints = np.random.randint(0, n, n)
+    return stat_fn(broadcast_data.value[randints])
+
+
+def run_bootstrap(sc, data, stat_fn, num_iterations=1000, confidence_level=0.95):
+    if not type(data) == np.ndarray:
+        data = np.array(data)
+
+    # Broadcast the data as read-only to the clusters.
+    broadcast_data = sc.broadcast(data)
+
+    f = partial(bootstrap, stat_fn=stat_fn, broadcast_data=broadcast_data)
+    stats = sc.parallelize(range(num_iterations)).map(f).collect()
+
+    broadcast_data.unpersist()
+
+    # Given the distribution of stats, calculate the percentiles based on the
```

Comment:
Actually, one more thought: consider abstracting getting the confidence intervals given a list of pseudostats into another method called `_ci_percentile` -- this way, if we decide to implement bca or some other algorithm we can make it easy to choose one of those as a parameter